package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "types": "colorette.d.ts",
   "scripts": {
-    "test": "nyc -r lcov testmatrix test/*.js",
+    "test": "nyc -r lcov testmatrix test/index.js",
```

Comment:
Good question because escaping the glob string using `'test/*.js'` does not seem to work and fails on all platforms.

Reply:
Probably because it is not directly glob related but normal `*` unix placeholder. Would be `test/**/*.js` ok?

Reply:
> Probably because it is not directly glob related but normal * unix placeholder.

Ah, that makes sense. So, I guess one would use a npm module to expand the glob, similarly to how one would use `rimraf` instead of `rm -rf`, and so on.

> Would be test/**/*.js ok (if it works)?

If `test/index.js` works, just leave it to that :)